Makefile

```diff
@@ -0,0 +1,23 @@
+install-matcher:
+	kubectl apply -f pattern_matcher/resources.yaml
+
```

Comment:
Please add a Make All and assume its being run from the path where the Makefile is located